src/index.ts

```diff
@@ -125,11 +125,8 @@ export class EIDRConnector extends BaseConnector {
 
     function validate(opt: string): string {
       if (typeof options[opt] !== 'string' || options[opt].length === 0) {
-        throw new EIDRError(
-          'Invalid ${opt}',
-          500,
-          `Invalid ${opt}: ${options[opt]}`
-        )
+        throw new EIDRError('Invalid ${opt}', 500,
+          `Invalid ${opt}: ${options[opt]}`)
```

Comment:
Why un-prettify (here and many places..)

Reply:
It was very verbose